Apps/WebClient/src/ClientApp/src/views/ServicesView.vue

```diff
@@ -0,0 +1,46 @@
+﻿<script lang="ts">
+import Vue from "vue";
+import { Component } from "vue-property-decorator";
+
+import LoadingComponent from "@/components/LoadingComponent.vue";
+import BreadcrumbComponent from "@/components/navmenu/BreadcrumbComponent.vue";
+import BreadcrumbItem from "@/models/breadcrumbItem";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const options: any = {
+    components: {
+        BreadcrumbComponent,
+        LoadingComponent,
+    },
+};
+@Component(options)
+export default class ServicesView extends Vue {
+    private isLoading = true;
+    private breadcrumbItems: BreadcrumbItem[] = [
+        {
+            text: "Services",
+            to: "/services",
+            active: true,
+            dataTestId: "breadcrumb-services",
+        },
+    ];
```

Comment:
The `private` modifier on getters and properties in Vue components should be removed as we add new files and update old ones, to prevent Volar reporting problems of the form:

`Property 'breadcrumbItems' is private and only accessible within class 'ServicesView'`